src/app/(main)/[locale]/layout.tsx

```diff
@@ -54,6 +56,7 @@ export default async function Layout({
     initialSoMe,
     initialLegal,
     initialCompanyLocations,
+    initialColorPallette,
```

Comment:
```suggestion
    initialColorPalette,
```
And similarly for the other places where we have pallette instead of palette. Easy mistake to make!